src/sagemaker/jumpstart/exceptions.py

```diff
@@ -58,6 +58,32 @@ def get_wildcard_model_version_msg(
     )
 
 
+def get_proprietary_model_subscription_msg(
+    model_id: str,
+    subscription_link: str,
+) -> str:
+    """Returns customer-facing message for using a proprietary model."""
+
+    return (
+        f"INFO: Using proprietary model '{model_id}'. "
+        f"Please make sure to subscribe to the model from {subscription_link}"
+    )
```

Comment:
non-blocking: review with tech writer please, `from <link>` sounds weird to me

Reply:
Updated to `Please make sure to subscribe to the model on {subscription_link}`